card/gem_mod/card-mod-ace_editor/card-mod-ace_editor.gemspec

```diff
@@ -20,4 +20,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.3.0"
   s.metadata = { "card-mod" => "ace_editor" }
   s.add_runtime_dependency "card-mod-edit", Versioning.simple
+  s.add_runtime_dependency "card", Versioning.card
```

Comment:
I guess this is redundant...

Reply:
right, if you depend on another card-mod-gem, it likely already has card in the chain of deps. So either on card or another card-mod- is enough.

Reply:
fixed